internals/secrethub/service_gcp_link.go

```diff
@@ -0,0 +1,193 @@
+	clause.Arg("project-id", "The GCP project to link the namespace to.").Required().StringVar(&cmd.projectID)
+
+	command.BindAction(clause, cmd.Run)
+}
+
+// ServiceGCPListLinksCommand lists all existing links between the given namespace and GCP projects
+type ServiceGCPListLinksCommand struct {
+	namespace     string
+	useTimestamps bool
+	io            ui.IO
+	newClient     newClientFunc
+}
+
+func NewServiceGCPListLinksCommand(io ui.IO, newClient newClientFunc) *ServiceGCPListLinksCommand {
+	return &ServiceGCPListLinksCommand{
+		io:        io,
+		newClient: newClient,
+	}
+}
+
+func (cmd *ServiceGCPListLinksCommand) Run() error {
+	timeFormatter := NewTimeFormatter(cmd.useTimestamps)
+
+	client, err := cmd.newClient()
+	if err != nil {
+		return err
+	}
+
+	links, err := client.IDPLinks().GCP().List(cmd.namespace)
+	if err != nil {
+		return err
+	}
+
+	f := newTableFormatter(cmd.io.Output(), 80, []tableColumn{
+		{name: "Project ID"}, {name: "Created at", maxWidth: 22},
+	})
+	for _, link := range links {
+		err := f.Write([]string{link.LinkedID, timeFormatter.Format(link.CreatedAt)})
+		if err != nil {
+			return err
+		}
+	}
+
+	return nil
+}
+
+func (cmd *ServiceGCPListLinksCommand) Register(r command.Registerer) {
+	clause := r.Command("list-links", "List all existing links between the given namespace and GCP projects.")
+	clause.Arg("namespace", "The namespace for which to list all existing links to GCP projects.").Required().StringVar(&cmd.namespace)
+
+	registerTimestampFlag(clause).BoolVar(&cmd.useTimestamps)
+
+	command.BindAction(clause, cmd.Run)
+}
+
+// ServiceGCPDeleteLinkCommand deletes the link between a SecretHub namespace and a GCP project.
+type ServiceGCPDeleteLinkCommand struct {
+	namespace string
+	projectID string
+	io        ui.IO
+	newClient newClientFunc
+}
+
+func NewServiceGCPDeleteLinkCommand(io ui.IO, newClient newClientFunc) *ServiceGCPDeleteLinkCommand {
+	return &ServiceGCPDeleteLinkCommand{
+		io:        io,
+		newClient: newClient,
+	}
+}
+
+func (cmd *ServiceGCPDeleteLinkCommand) Register(r command.Registerer) {
+	clause := r.Command("delete-link", "Delete the link between a SecretHub namespace and a GCP project.")
+	clause.Arg("namespace", "The SecretHub namespace to delete the link from.").Required().StringVar(&cmd.namespace)
+	clause.Arg("project-id", "The GCP project to delete the link to.").Required().StringVar(&cmd.projectID)
+
+	command.BindAction(clause, cmd.Run)
+}
+
+func (cmd *ServiceGCPDeleteLinkCommand) Run() error {
+	client, err := cmd.newClient()
+	if err != nil {
+		return err
+	}
+
+	_, err = client.IDPLinks().GCP().Get(cmd.namespace, cmd.projectID)
+	if api.IsErrNotFound(err) {
+		return err
+	}
+
+	question := fmt.Sprintf("Are you sure you want to delete the link link between GCP project %s and the namespace %s? This does not affect any existing service accounts.", cmd.projectID, cmd.namespace)
```

Comment:
Could be helpful to describe the consequence of deleting the link: 
```suggestion
	question := fmt.Sprintf("Are you sure you want to delete the link link between GCP project %s and the namespace %s? Without the link, you cannot create new service accounts for this GCP project. This does not affect existing service accounts.", cmd.projectID, cmd.namespace)
```